apps/dav/lib/CalDAV/Schedule/IMipPlugin.php

```diff
@@ -265,12 +265,14 @@ public function schedule(Message $iTipMessage) {
 		$mailService = null;
 
 		try {
-			// retrieve user object
-			$user = $this->userSession->getUser();
-			// evaluate if user object exist
-			if ($user !== null) {
-				// retrieve appropriate service with the same address as sender
-				$mailService = $this->mailManager->findServiceByAddress($user->getUID(), $sender);
+			if ($this->config->getSystemValueInt('mail_providers_disabled', 0) === 0) {
```

Comment:
```suggestion
			if ($this->config->getSystemValueInt('mail_providers_disabled', 0) === 0) {
```

Must be documented in https://github.com/nextcloud/server/blob/master/config/config.sample.php.

https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/config_sample_php_parameters.html is also generated from the config.sample.php.